bin/mysql-client-admin

```diff
@@ -5,11 +5,13 @@ require 'uri'
 
 raise 'please define CDO.db_credential_admin' unless CDO.db_credential_admin
 credentials = CDO.db_credential_admin
+host = CDO.db_endpoint_writer.split(':')[0]
+port = CDO.db_endpoint_writer.split(':')[1] || 3306
 db_connection_settings = URI::Generic.build(
   scheme: 'mysql',
   userinfo: credentials['username'] + ':' + credentials['password'],
-  host: CDO.db_endpoint_writer,
-  port: 3306,
+  host: host,
+  port: port,
```

Comment:
Happy to see this change, its not really best practice to run services from docker-compose binding to the default port for that service (because you have to add notes like:
> If you are already running MySQL and/or Redis natively, you may depending on your
> operating system need to stop those services"